### PR TITLE
Grafana 4 cosmetic adaptations + config panel

### PR DESCRIPTION
**Pull request by samuelbenhamou**

### CODE REVIEW

File: config_ctrl.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+/// <reference path="/usr/share/grafana/public/app/headers/common.d.ts" />
```
Comment (jifwin) [inline config_ctrl.ts]: This should definitely be a relative path. Is it really needed? Actually I made it work without imports and reference. Please remove those.

**Comment by jifwin** [comment]
Thank you for this PR, it makes good progress in UI. However it still needs some polish. PRs welcomed :).

File: config_ctrl.ts (hunk 1 above)
Comment (jifwin) [inline config_ctrl.ts]: I'm going to clean this up myself in seperate branch. Thank you.

**Comment by samuelbenhamou** [comment]
sorry did not have much time to look at it. Great you did the cleanup. I'll test the new branch and check what's not working

**Pull request merged**